dlog/commitment/src/srs.rs

```diff
@@ -22,6 +22,40 @@ pub struct SRS<G: CommitmentCurve> {
     pub endo_q: G::BaseField,
 }
 
+pub struct TrimmedSRS<'a, G: CommitmentCurve> {
+    pub srs: &'a SRS<G>,
+    pub trimmed_length_log2: usize
+}
```

Comment:
I'd be good to comment anything public so we can generate some rustdoc at some point :o